packages/app/src/components/DBTraceWaterfallChart.tsx

```diff
@@ -61,10 +58,45 @@ function getTableBody(tableModel: TSource) {
   }
 }
 
-function getConfig(source: TSource, traceId: string) {
+function useSortKey(source: TSource): string {
+  let rowSortTimestamp = source.timestampValueExpression;
+  const { data: sourceCols } = useColumns({
```

Comment:
I think we want to use `useAllFields` (https://github.com/hyperdxio/hyperdx/blob/e80630c1075d381f91dde43301dfceb92b95100e/packages/app/src/hooks/useMetadata.tsx#L38) instead